--- python-driver-3.7.1/setup.py~	2016-10-26 17:00:31.000000000 +0200
+++ python-driver-3.7.1/setup.py	2016-12-29 22:42:29.223605087 +0100
@@ -388,7 +388,7 @@ def run_setup(extensions):
         # 1.) build_ext eats errors at compile time, letting the install complete while producing useful feedback
         # 2.) there could be a case where the python environment has cython installed but the system doesn't have build tools
         if pre_build_check():
-            kw['setup_requires'] = ['Cython>=0.20,<0.25']
+            kw['setup_requires'] = ['Cython>=0.20,<0.26']
         else:
             sys.stderr.write("Bypassing Cython setup requirement\n")
 
